es/07_2_Usando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md

```diff
@@ -0,0 +1,593 @@
+          "master_fingerprint": "be686772",
+          "path": "m"
+        },
+        {
+          "pubkey": "038d73adf2c7ea33f9dc34b77b62b59af433c1de9c763332da79e83e155f96030e",
+          "master_fingerprint": "d6043800",
+          "path": "m/0'/0'/26'"
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "bip32_derivs": [
+        {
+          "pubkey": "02fce26085452d07abc63bd389cb7dba9871e79bbecd08039291226be8232a9000",
+          "master_fingerprint": "d6043800",
+          "path": "m/0'/0'/24'"
+        }
+      ]
+    },
+    {
+      "bip32_derivs": [
+        {
+          "pubkey": "0349cc43324f7ad94bb407a9bf12bc50afd9e7b430a472572f1b63cb555034f52a",
+          "master_fingerprint": "d3ed8825",
+          "path": "m/0'/0'/3'"
+        }
+      ]
+    }
+  ],
+  "fee": 0.00000200
+}
+$ bitcoin-cli analyzepsbt $psbt_c
+{
+  "inputs": [
+    {
+      "has_utxo": true,
+      "is_final": false,
+      "next": "finalizer"
+    }
+  ],
+  "estimated_vsize": 168,
+  "estimated_feerate": 0.00001190,
+  "fee": 0.00000200,
+  "next": "finalizer"
+}
+```
+¡Funcionó! Simplemente finalizamos y enviamos y terminamos:
+```
+machine2$ psbt_c_hex=$(bitcoin-cli finalizepsbt $psbt_c | jq -r '.hex')
+standup@btctest2:~$ bitcoin-cli -named sendrawtransaction hexstring=$psbt_c_hex
+ee82d3e0d225e0fb919130d68c5052b6e3c362c866acc54d89af975330bb4d16
+```
+Obviamente, no hubo una gran mejora en el uso de este método en comparación con la firma en serie de una transacción para una multifirma 2 de 2 cuando todos usaban `bitcoin-cli`: podríamos haber pasado una transacción sin procesar con firmas parciales de un usuario a otro tan fácilmente como enviar ese PSBT. Pero este fue el caso más simple. A medida que nos adentramos en multifirmas más complejas, esta metodología se vuelve cada vez mejor.
+
+En primer lugar, es independiente de la plataforma. Siempre que todos utilicen un servicio que admita Bitcoin Core 0.17, todos podrán firmar esta transacción, lo cual no es cierto cuando los multifirmas clásicos se transmiten entre diferentes plataformas.
+
+En segundo lugar, es mucho más escalable. Considere un multifirma de 3 de 5. Bajo la vieja metodología, tendría que pasar de persona a persona, aumentando enormemente los problemas si se rompe algún eslabón de la cadena. Aquí, otros usuarios solo tienen que enviar los PSBT al Creador, y tan pronto como tenga suficiente, puede generar la transacción final.
+
+## Utilice un PSBT para juntar dinero
+
+Multifirmas como la utilizada en el ejemplo anterior se suelen utilizar para recibir pagos por trabajo colaborativo, ya sean regalías por un libro o pagos realizados a una empresa. En esa situación, el ejemplo anterior funciona muy bien: los dos participantes reciben su dinero y luego lo dividen. Pero, ¿qué pasa con el caso inverso, donde dos (o más) participantes quieren establecer una empresa conjunta y necesitan sembrarla con dinero?
```

Comment:
I think that `necesitan financiarla con dinero?` is better. But `sembrarla` is not bad.